src/pages/api/definePassengerType.ts

```diff
@@ -159,6 +166,9 @@ export const formatRequestBody = (req: NextApiRequestWithSession): FilteredReque
             return;
         }
         if (entry[0] === 'proofDocuments') {
```

Comment:
Could we change this to one line? `if (entry[0] === 'proofDocuments' && req.body.proof !== 'No') {
            filteredReqBody[entry[0]] = !isArray(entry[1]) ? [entry[1] as string] : (entry[1] as string[]);
            return;
        }`

Reply:
Ah, so the problem is that we need to return if we have the combination of `entry[0] === 'proofDocuments' && req.body.proof !== 'No'`, as otherwise line 175 will execute. I do agree it could be simplified though, how about this:
```
if (entry[0] === 'proofDocuments') {
    if (req.body.proof === 'Yes') {
        filteredReqBody[entry[0]] = !isArray(entry[1]) ? [entry[1] as string] : (entry[1] as string[]);
    }
    return;
}
```

Reply:
Yeah sounds good!